### PR TITLE
Fix FQDN in for the middle tier SSL cert.

### PR DESCRIPTION
**Pull request by ivan-shaporov**
The FQDN in the SSL must match FQDN of the Azure Traffic Manager End Point. And the endpoint must point to the traefik load balancer not to the management endpoint of the k8s cluster.

\+ some cosmetic changes making troubleshooting the script easier.

### CODE REVIEW

File: deployment/createMtSvc.sh [hunk 1]
```diff
@@ -47,6 +47,7 @@ az keyvault secret set --name token-sign-key  --vault-name $MT_KEYVAULT_NAME --d
 
 # use self signed cert for now
 SSL_PASSWORD=`uuidgen`
+FQDN=`az network traffic-manager profile list --resource-group $COMMON_RESOURCE_GROUP --query [0].dnsConfig.fqdn -o tsv`
```
Comment (wesyao) [inline deployment/createMtSvc.sh]: might be beneficial to write this into the cluster info file that is generated?

Comment (aaron-schnieder) [reply deployment/createMtSvc.sh]: +1

**Comment by aaron-schnieder** [comment]
Agreed. Ivan, could you add a description with PRs to give reviewers an overview of the changes being made?

File: deployment/createMtSvc.sh (hunk 1 above)
Comment (ivan-shaporov) [reply deployment/createMtSvc.sh]: where is that file generated?

Comment (billpratt) [reply deployment/createMtSvc.sh]: @wesyao are you talking about the ```inception.txt``` that is generated?

Comment (wesyao) [reply deployment/createMtSvc.sh]: @billpratt I was referring to `inception.txt`

**Pull request merged**